cider/exec/plan/parser/SubstraitToAnalyzerExpr.cpp

```diff
@@ -92,8 +93,8 @@ std::shared_ptr<Analyzer::Expr> Substrait2AnalyzerExprConverter::toAnalyzerExpr(
     case substrait::Expression::RexTypeCase::kIfThen:
       return toAnalyzerExpr(s_expr.if_then(), function_map, expr_map_ptr);
     default:
-      throw std::runtime_error("Unsupported expression type " +
-                               std::to_string(s_expr.rex_type_case()));
+      CIDER_THROW(CiderRuntimeException,
```

Comment:
should not be CiderRuntimeException. cc @yma11

Reply:
fix